scripts/common.sh

```diff
@@ -46,3 +46,15 @@ function remove_kmod() {
     sudo rmmod $mod_name > /dev/null
     return 0
 }
+
+function start_hostapd() {
+	echo "Start Hostapd"
+	sudo hostapd -B $1 > /dev/null
```

Comment:
Use `which` to check the existence of `hostapd` command.

Reply:
done